code/modules/reagents/chemistry/reagents/medicine.dm

```diff
@@ -686,6 +686,9 @@
 	color = "#5096C8"
 
 /datum/reagent/medicine/mutadone/on_mob_life(mob/living/carbon/human/M)
+	if(M.mind && M.mind.assigned_role == "Cluwne") // HUNKE
```

Comment:
This is disgusting and snowflakey, but the alternative was to delete mutadone by the honk tumor every `on_life` *and* re-add the mutations that it removed, which is expensive; it's better, performance and cleanliness wise, to just have this here.